### DISCUSSION TITLE
Journalists Wanted for The Bumpkin News!

### DISCUSSION THREAD

**Discussion by drumdrumdrum3337** [sunflower-land/sunflower-land#1716] [General]
Do you love arranging words in order to create a compelling story?
Do your fingers itch to type something meaningful instead of "Lorem ipsum" wherever you see the placeholder text?
Do you think Clark Kent was cool because he was working in a newspaper, and not because he could fly and lift a Titanic?

If yes, then read on!

The very first in-game newspaper in the world, The Bumpkin News, is looking for journalists! 

![front by Boden](https://user-images.githubusercontent.com/108594866/202829272-33d6a13b-d14a-4a59-b9ab-6cada76eb481.png)

Do you feel like reporting back to the community from Adam's weekly streams is what you'd like to do? Or do you enjoy browsing discord and would love to share some thoughts and observations you found there with the readers? Or you are more into writing longreads, essays and taking interviews? Maybe a quiz creation is your thing? 

If you've responded 'Yes!" to any of these, then go ahead and apply!

To join us on this journey through the uncharted territories of in-game media, please submit the following:

- A bit about yourself: totally freestyle and include whatever you believe could be relevant.
- A brief essay about how you see The Bumpkin News develop in the future, and what could be your role in it;
- Ideally, but not mandatory: an example of your other related work, such as links to your existing articles, quizzes created, etc.

Currently, the team includes Tiffany, who is taking care of the coding part (she's also part of the Project Dignity), and Jc and Boden as artists.

Important notes:

- the newspaper is planned to go out every two weeks, hence this project will require time investment from you;
- this role is not a volunteer one - all members of the team will be paid from profits generated from the sales of the newspaper;
- Ideally we'd love 3-5 journalists to join the editorial team; 

Please feel free to post your submissions in this discussion, reach out on the discord, or just send us an email to [the-bumpkin-news@googlegroups.com](mailto:the-bumpkin-news@googlegroups.com)

The original newspaper proposal [is here](https://github.com/sunflower-land/sunflower-land/discussions/1367)

**Comment by sprmwhale5183** [comment]
Hello there! I'm based out of Malaysia and when I really do have free time I enjoy playing ultimate frisbee. In my day job I do a lot of writing, mainly for corporations engaging with their internal audiences. While I haven't had a lot of experience being an actual journalist per se, I've done internal newsletters, preparations for townhalls and generally keeping an internal communications platform engaging for a corporate setting.

I see Bumpkin News taking on 2 prongs: 
1. Keeping the community informed and connected. (Between devs, players and among themselves too) 
2. Another platform for engaging with the community.

On the first prong, not all community members may be available to attend live streams / sessions and a tldr column that summarises the multiple sessions held could prove to be beneficial. 

We should also use it to highlight the wider team such as discord mods, illustrators, and even community members who've done well for the community.

I also think it's a good opportunity for us to highlight the charities that people invest in when minting their farm initially. Would be cool to see how they've done with the contributions.

Pulling all this content together, it's totally possible to use the newsletters for more in game utility or clues such as piecing together a treasure map from various parts of 4 newsletters. I have held amateurish text based treasure hunts before in my current workplace which required participants to comb through our intranet but nothing like an article per se.

Hit me up if I seem like a right fit!
Discord: sprmwhale5183

**Reply by drumdrumdrum3337** [reply]
Thank you for the submission! We'll keep the call open for 2-3 weeks for folks to apply and then post the selected team here. 

Note for other folks: it's also fine to send all that is required to the email mentioned in the post itself ([the-bumpkin-news@googlegroups.com](mailto:the-bumpkin-news@googlegroups.com))
